File: Odd_or_Even.java
import java.util.Scanner;
import java.lang.String;
/**
 * Write a description of class odd_or_even here.
 *
 * @author (your name)
 * @version (a version number or a date)
 */
public class Odd_or_Even
{
    static String odd_or_even(int num){
        if(num % 2 == 0){
            return "even";
        }
        else{
            return "odd";
        }
    }
    
    
    public static void main(String[]args){
        Scanner input = new Scanner(System.in);
        
        System.out.println("Enter number: ");
        int num = input.nextInt();
        System.out.println(odd_or_even(num));
    }
}

File: odd_or_even.js
odd_or_even = (number) => {
    if (number % 2 === 0) {
        return "even";
    } else {
        return "odd";
    }
}


var input = require('readline');
var prompts = input.createInterface(process.stdin,process.stdout);

prompts.question("Enter a number:  ",(number) =>{
    console.log(odd_or_even(number));
    process.exit();
});